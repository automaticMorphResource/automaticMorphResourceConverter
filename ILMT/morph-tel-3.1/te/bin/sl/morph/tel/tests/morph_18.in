1	kaMgArU	unk	
1	xIvi	unk
1	...	unk	
1	ixi	unk	
1	eRiyA	unk	
1	PasiPik	unk	
1	prAMwaMlo	unk	
1	uMde	unk	
1	anni	unk	
1	xIvulaloki	unk
1	allA	unk	
1	aMxamEnaxigA	unk	
1	perugAMciMxi	unk	
1	.	unk	
