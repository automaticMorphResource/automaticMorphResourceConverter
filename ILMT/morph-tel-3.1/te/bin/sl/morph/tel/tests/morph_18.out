1	kaMgArU	unk	<fs af='kaMgArU,n,,sg,,d,0,0'>|<fs af='kaMgArU,n,,sg,,o,ti,ti'>|<fs af='kaMgAru,n,,sg,,,0,0_V'>
2	xIvi	unk	<fs af='xIvi,n,,sg,,d,0,0'>|<fs af='xIvi,n,,sg,,o,ti,ti'>
3	...	unk	<fs af='..,punc,,,,,,'>
4	ixi	unk	<fs af='ixi,pn,fn,sg,3,d,0,0'>
5	eRiyA	unk	<fs af='eRiyA,n,,sg,,d,0,0'>|<fs af='eRiyA,n,,sg,,o,ti,ti'>
6	PasiPik	unk	<fs af='PasiPik,n,,sg,,d,0,0'>|<fs af='PasiPik,n,,sg,,o,ti,ti'>
7	prAMwaMlo	unk	<fs af='prAMwaM,n,,sg,,,lo,lo'>
8	uMde	unk	<fs af='uMdu,v,any,any,any,,e,e'>|<fs af='uMdu,v,any,any,any,,i,i_e'>
9	anni	unk	<fs af='anni,pn,,pl,,d,0,0'>
10	xIvulaloki	unk	<fs af='xIvi,n,,pl,,,lopalaki,lopalaki'>
11	allA	unk	<fs af='allA,avy,,,,,,'>|<fs af='allu,v,any,any,any,,i_A,i_A'>
12	aMxamEnaxigA	unk	<fs af='aMxaM,pn,,sg,,,Ena_xi,Ena_xi_gA'>
13	perugAMciMxi	unk	<fs af='perugAMcu,v,fn,sg,3,,A,A'>
14	.	unk	<fs af='&dot;,punc,,,,,,'>
